File: app/src/main/java/com/harsha/data/model/TopPicks.java
package com.harsha.data.model;

import android.os.Parcel;
import android.os.Parcelable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class TopPicks implements Parcelable
{
    @SerializedName("drills")
    @Expose
    private ArrayList<Drills> drills;

    @SerializedName("title")
    @Expose
    private String title;

    protected TopPicks(Parcel in) {
        title = in.readString();
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeString(title);
    }

    @Override
    public int describeContents() {
        return 0;
    }

    public static final Creator<TopPicks> CREATOR = new Creator<TopPicks>() {
        @Override
        public TopPicks createFromParcel(Parcel in) {
            return new TopPicks(in);
        }

        @Override
        public TopPicks[] newArray(int size) {
            return new TopPicks[size];
        }
    };

    public ArrayList<Drills> getDrills ()
    {
        return drills;
    }

    public void setDrills (ArrayList<Drills> drills)
    {
        this.drills = drills;
    }

    public String getTitle ()
    {
        return title;
    }

    public void setTitle (String title)
    {
        this.title = title;
    }

    @Override
    public String toString()
    {
        return "ClassPojo [drills = "+drills+", title = "+title+"]";
    }
}

File: app/src/main/java/com/harsha/data/model/TopPicks.kt
package com.harsha.data.model

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TopPicks() :Parcelable {
    @SerializedName("drills")
    @Expose
    private var drills: ArrayList<Drills>? = null

    @SerializedName("title")
    @Expose
    private var title: String? = null

    constructor(parcel: Parcel) : this() {
        title = parcel.readString()
    }

    fun getDrills(): ArrayList<Drills>? {
        return drills
    }

    fun setDrills(drills: ArrayList<Drills>?) {
        this.drills = drills
    }

    fun getTitle(): String? {
        return title
    }

    fun setTitle(title: String?) {
        this.title = title
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(title)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<TopPicks> {
        override fun createFromParcel(parcel: Parcel): TopPicks {
            return TopPicks(parcel)
        }

        override fun newArray(size: Int): Array<TopPicks?> {
            return arrayOfNulls(size)
        }
    }
}